recipes/ios-cmake/all/conanfile.py

```diff
@@ -0,0 +1,89 @@
+
+from conans import ConanFile, tools
```

Comment:
```suggestion
from conans import ConanFile, tools
from conans.errors import ConanInvalidConfiguration
```

Reply:
currently I do not throw any error , shall I do the import anyway?

Reply:
no, remove unused imports to avoid warnings

Reply:
I added it for https://github.com/conan-io/conan-center-index/pull/1137/files#r394701550

But I changed it from a suggestion to a comment.